packages/gestalt/src/Touchable.js

```diff
@@ -73,126 +84,162 @@ const getRoundingStyle = (r: Rounding): Style => {
   return identity();
 };
 
-export default class Touchable extends React.Component<Props> {
-  static propTypes = {
-    accessibilityControls: PropTypes.string,
-    accessibilityExpanded: PropTypes.bool,
-    accessibilityHaspopup: PropTypes.bool,
-    accessibilityLabel: PropTypes.string,
-    children: PropTypes.node,
-    disabled: PropTypes.bool,
-    fullHeight: PropTypes.bool,
-    fullWidth: PropTypes.bool,
-    mouseCursor: PropTypes.oneOf([
-      'copy',
-      'grab',
-      'grabbing',
-      'move',
-      'noDrop',
-      'pointer',
-      'zoomIn',
-      'zoomOut',
-    ]),
-    onBlur: PropTypes.func,
-    onFocus: PropTypes.func,
-    onTouch: PropTypes.func,
-    onMouseEnter: PropTypes.func,
-    onMouseLeave: PropTypes.func,
-    rounding: RoundingPropType,
-  };
-
-  handleKeyPress = (event: SyntheticKeyboardEvent<HTMLDivElement>) => {
-    const { disabled, onTouch } = this.props;
-    // Check to see if space or enter were pressed
-    if (
-      !disabled &&
-      onTouch &&
-      (event.charCode === SPACE_CHAR_CODE || event.charCode === ENTER_CHAR_CODE)
-    ) {
-      // Prevent the default action to stop scrolling when space is pressed
-      event.preventDefault();
-      onTouch({ event });
-    }
-  };
-
-  handleBlur = (event: SyntheticFocusEvent<HTMLDivElement>) => {
-    const { disabled, onBlur } = this.props;
-    if (!disabled && onBlur) {
-      onBlur({ event });
-    }
-  };
-
-  handleFocus = (event: SyntheticFocusEvent<HTMLDivElement>) => {
-    const { disabled, onFocus } = this.props;
-    if (!disabled && onFocus) {
-      onFocus({ event });
+function Touchable({
+  accessibilityControls,
+  accessibilityExpanded,
+  accessibilityHaspopup,
+  accessibilityLabel,
+  children,
+  disabled = false,
+  forwardedRef,
+  fullHeight,
+  fullWidth = true,
+  mouseCursor = 'pointer',
+  onBlur,
+  onFocus,
+  onMouseEnter,
+  onMouseLeave,
+  onTouch,
```

Comment:
okay for a followup - does this make sense to leave as `onTouch` if we're going to rename this to `TapArea` eventually?  maybe `onTap` instead?

Reply:
I think we can leave as is and rename when we are ready to codemod everything to `TapArea`. This way we can just codemod both component name and this prop in one diff instead of two.